mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/generic_reduce_2d.mlir

```diff
@@ -0,0 +1,95 @@
+// DEFINE: %{compile} =  mlir-opt %s \
+// DEFINE:    -transform-interpreter -test-transform-dialect-erase-schedule \
+// DEFINE:    -one-shot-bufferize="bufferize-function-boundaries" -buffer-deallocation-pipeline -cse -canonicalize -convert-vector-to-scf -arm-sve-legalize-vector-storage \
+// DEFINE:    -convert-vector-to-llvm="enable-arm-sve" -test-lower-to-llvm -o %t
+// DEFINE: %{entry_point} = generic_reduce_2d_f32
+// DEFINE: %{run} = %mcr_aarch64_cmd %t -e %{entry_point} -entry-point-result=void --march=aarch64 --mattr="+sve"\
+// DEFINE:    -shared-libs=%mlir_native_utils_lib_dir/libmlir_runner_utils%shlibext,%mlir_native_utils_lib_dir/libmlir_c_runner_utils%shlibext
```

Comment:
`%mlir_native_utils_lib_dir/libmlir_runner_utils%shlibext` will expands to `config.arm_emulator_utils_lib_dir/libmlir_runner_utils%shlibext`.

config.arm_emulator_utils_lib_dir is defined in https://github.com/llvm/llvm-project/blob/main/mlir/test/lit.site.cfg.py.in#L59